### PR TITLE
Add pre-submit Kyma Lite on kind job

### PR DESCRIPTION
**Pull request by michal-hudy**
**Description**

Changes proposed in this pull request:

- Added scripts
- Added job definition
- Notifications to GH are disabled
- Scripts are working also on a local computer, not only in the Prow environment

As this is the first attempt, so documentation is missing and for sure Kyma Lite is supported.

**Related issue(s)**

#1654

### CODE REVIEW

File: prow/scripts/lib/kyma.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```
Comment (dekiel) [inline prow/scripts/lib/kyma.sh]: Please add description of function arguments.

File: prow/scripts/lib/kind.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```
Comment (dekiel) [inline prow/scripts/lib/kind.sh]: Please add description of function arguments.

**Comment by michal-hudy** [comment]
Sample job result: https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/6122/pre-master-kyma-kind-lite/1188802214928322560

@dekiel As I said in PR description, this is just first attempt, and documentation is missing. During migration of full Kyma I have a plan to add proper comments and documentation, because the library should be stable at that point. Of course, if you think that documentation is really needed at the moment I can add it 😄

**Comment by dekiel** [comment]
@michal-hudy It's perfectly fine to add comments later when you consider library is ready for use for others. My intention was only to give feedback, please don't consider this as a blocker from merging.

**Pull request merged**